packages/validator/templates/1.0.0/endpoints.json

```diff
@@ -0,0 +1,71 @@
+{
+  "__arrayItem": {
+    "name": {},
```

Comment:
Could anything bad happen if we don't have regex to check for certain characters?

Reply:
There is no requirement in the documentation, but it might be worth to ask people working on integrations to try submit bunch of invalid specifications to see if validator covers all edge cases.